std/digest/hmac.d

```diff
@@ -29,18 +31,19 @@ import std.meta : allSatisfy;
  * This type conforms to $(REF isDigest, std,digest,digest).
  */
 
-version(StdDdoc)
-/// Computes an HMAC over data read from stdin.
+/// Compute HMAC over an input string
 @safe unittest
 {
-    import std.digest.hmac, std.digest.sha, std.stdio;
+    import std.ascii : LetterCase;
+    import std.digest.digest : toHexString;
```

Comment:
#5013 was merged and as far as I understand, this can be changed to: `import std.digest : toHexString`